external-docs/docs/guides/configuration/qube-config/columns/units.md

```diff
@@ -0,0 +1,145 @@
+# Units columns
+
+This page discusses what a units column is, where one should be used, and how one can be defined.
+
+The configuration of unit definitions themselves will not be the primary focus of this page; for help with this, see
+[configuring units](./units.md).
+
+> For a detailed look at a unit column's configuration options, see the [Reference table](#reference) at the bottom of
+>this page.
+
+## What is a units column?
+
+A *units* column describes which unit each observed value has been measured in.
+
+The following is an example of a small data set containing a units column:
+
+| Year | Location      | Average Height of Men |        Unit |
+|:-----|:--------------|----------------------:|------------:|
+| 2019 | Canada        |                    70 |      Inches |
+| 2020 | United States |                    69 |      Inches |
+| 2021 | England       |                   175 | Centimetres |
+
+The units column is titled `Unit`. The first two rows of the data set use the unit `Inches` to measure height, and the
+third row uses a different unit, `Centimetres`. In this case, different units are being used to measure the same thing.
+
+## When to use a units column
+
+Every valid data cube needs at least one unit. Units columns can be used in both
+[standard](../../../shape-data/standard-shape.md) and [pivoted shape](../../../shape-data/pivoted-shape.md) cubes.
+
+If every value in an observations column has the same unit, then you should set the unit against the
+[observations column](./observations.md).
+
+If some values in your observations column use one unit and some other values in the same column use a different unit
+then you should use a units column. In the example table above, we see a clear example of this given that the first two
+rows use `Inches` and the third row uses `Centimetres` to measure the `Average Height of Men`.
+
+## Basic configuration
+
+The following JSON shows how a units column can be defined in a [qube configuration file](../index.md):
+
+```json
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Average Height of Men in different countries",
+    "columns": {
+        "Unit": {
+            "type": "units"
+        },
+    }
+}
+```
+
+To define a units column, specify the `type` of the column definition as `units`.
+
+<!-- TODO: At some point, add an example of a single measure pivoted data set with a units column (multiple units) -->
+
+todo: write about default behaviour of units column configuration
```

Comment:
TODO

Reply:
I believe we now do this in the example data set above?